src/ode_solver/runge_kutta_methods/runge_kutta_methods.cpp

```diff
@@ -9,6 +9,8 @@ void SSPRK3Explicit<dim,real,MeshType> :: set_tableau()
 {
     set_a();
     set_b();
+    set_c();
+    this->pcout << "Assigned RK method: 3rd order SSP (explicit)" << std::endl;
```

Comment:
If the definition of this function does get moved to the base class, you could have a `const std::string rk_method_string_input` as an input argument to the `RKTableauBase` constructor for initializing member: `const std::string rk_method_string`; then in `set_tableau()` you have `set_a(); set_b(); set_c(); this->pcout << "Assigned RK method: " << this->rk_method_string << std::endl;`

Reply:
I've implemented this.